tests/integration/test_kube_ovn.py

```diff
@@ -210,14 +206,50 @@ async def test_pod_netem_limit(ops_test, kubeconfig, client, iperf3_pods):
     assert actual_limit == expected_limit
 
 
+async def test_gateway_qos(
+    ops_test, kubeconfig, client, gateway_server, gateway_client_pod, worker_node
+):
+    namespace = gateway_client_pod.metadata.namespace
+
+    rate_annotations = {
+        "ovn.kubernetes.io/ingress_rate": "60",
+        "ovn.kubernetes.io/egress_rate": "30",
+    }
+
+    await annotate_obj(client, worker_node, rate_annotations)
+
+    # We need to wait a little bit for OVN to do its thing
+    # after applying the annotations
+    time.sleep(30)
```

Comment:
you may consider using 

```suggestion
    await asyncio.sleep(30)
```